src/components/VMediaInfo/VCopyLicense.vue

```diff
@@ -41,13 +40,10 @@
           path="media-details.reuse.credit.text"
           tag="p"
         >
-          <template #title>
-            <VLink
-              :href="media.foreign_landing_url"
-              @click="onSourceLinkClicked"
-              @keyup.enter="onSourceLinkClicked"
-              >{{ media.title }}</VLink
-            ></template
+          <template #title
+            ><VLink :href="media.foreign_landing_url">{{
+              media.title
+            }}</VLink></template
           >
```

Comment:
The way prettier smashes Vue templates all into one line makes my brain hurt reading it :confounded: these lines are nearly incomprehensible to me without slowing down completely to figure out what's going on.

Reply:
same. i really dislike it

Reply:
I agree. I tried improving it, but wasn't very successful...